src/resources/assets/js/volumes/geoFilter.js

```diff
@@ -1,33 +1,49 @@
 import {VolumeFilters} from './import';
 import {FilterList} from './import';
-
+import FilterSelect from '../geo/components/filterByLocationComponent';
 /**
  * Geo filter for the volume overview filters.
  */
+
 if (Array.isArray(VolumeFilters)) {
     VolumeFilters.push({
-        id: 'geo',
+        id: 'location',
         types: ['image'],
-        label: 'geo selection',
+        label: 'filter by location',
         help: "All images that were (not) selected on the world map.",
         listComponent: {
             mixins: [FilterList],
             data() {
                 return {
-                    name: 'geo selection',
+                    name: 'filter by location',
                 };
             },
             created() {
+                // only triggered when called from different session/window
                 window.addEventListener('storage', () => {
                     this.$emit('refresh', this.rule);
                 });
+                // trigger rule-refresh, see FilterSelect-Component of this filter
+                window.addEventListener('storageUpdate', () => {
+                    this.$emit('refresh', this.rule);
+                });
```

Comment:
This may not be required (see above).

Reply:
resolved 6c7af8c877c9dfe1249ab10990be473a756e65ca